src/postgres/commands/PostgresAccountWizard/PostgresServerCreateStep.ts

```diff
@@ -18,14 +18,14 @@ export class PostgresServerCreateStep extends AzureWizardExecuteStep<IPostgresWi
 
         const locationName = nonNullProp(nonNullProp(wizardContext, 'location'), 'name');
         const rgName: string = nonNullProp(nonNullProp(wizardContext, 'resourceGroup'), 'name');
-        const serverName = nonNullProp(wizardContext, 'newServerName');
+        const serverName = nonNullProp(wizardContext, 'serverName');
```

Comment:
I believe we've discussed this before, but please continue to use `newServerName` like it was before instead of just `serverName`

Reply:
Do you mean just in this file or in IAzureDBWizardContext as well?

Reply:
In the wizard context, too. Basically everywhere